tests/unit/file_lib_test.c

```diff
@@ -27,6 +27,8 @@
 #include <file_lib.h>
 #include <stdbool.h>
 
+#include "../../libutils/file_lib.c"
```

Comment:
Why do we need this? If there are missing function declarations in the `file_lib.h` header, we should add them here.

Reply:
I don't remember right now. Removed it temporarily to remind me :wink:

Reply:
`safe_open()` does not work unless I include the file here or in the `file_lib_test_SOURCES` and I'm not smart enough to know why :thinking: 

The former causes problems with make target distclean on the bootstrap host when subdir_objects is enabled.

Reply:
Strange, but not worth spending extra time, AFAICT. :+1: